FWCore/Integration/test/run_SubProcess.sh

```diff
@@ -11,7 +11,7 @@ pushd ${LOCAL_TMP_DIR}
 
   echo cmsRun testSubProcess_cfg.py
   cmsRun -p ${LOCAL_TEST_DIR}/${test}_cfg.py >& ${test}.log 2>&1 || die "cmsRun ${test}_cfg.py" $?
-  grep Doodad ${test}.log > testSubProcess.grep.txt
+  grep Doodad ${test}.log | grep -v "Tracer" > testSubProcess.grep.txt
```

Comment:
This shouldn't be needed anymore.

Reply:
Agreed. First thing tomorrow I plan to rebase on top of the IB with the real fix for that problem and I'll remove my commit with that change at the same time. I already had it in a separate commit with the intent to remove it.